### PR TITLE
User permission and location display

### PR DESCRIPTION
**Pull request by DumitruBogdan**
In this branch, you can find the implementation of:
- requesting the user's permission for the location service
- displaying the geographical coordinates
- displaying the city name based on the determined coordinates

### CODE REVIEW

File: WeatherAPP/WeatherAPP/Features/Weather/View/WeatherView.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import CoreLocation
```
Comment (vladstanescu94) [inline WeatherAPP/WeatherAPP/Features/Weather/View/WeatherView.swift]: We don't need this import here

File: WeatherAPP/WeatherAPP/Util/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import CoreLocation
+import Foundation
+
```
Comment (vladstanescu94) [inline WeatherAPP/WeatherAPP/Util/ViewController.swift]: We don't need this file anymore

**Review by vladstanescu94 — CHANGES_REQUESTED** [review]

File: WeatherAPP/WeatherAPP/Util/ViewController.swift (hunk 2 above)
Comment (DumitruBogdan) [reply WeatherAPP/WeatherAPP/Util/ViewController.swift]: Resolved

File: WeatherAPP/WeatherAPP/Features/Weather/View/WeatherView.swift (hunk 1 above)
Comment (DumitruBogdan) [reply WeatherAPP/WeatherAPP/Features/Weather/View/WeatherView.swift]: Resolved

**Review by vladstanescu94 — APPROVED** [review]

**Review by mihaigpm — APPROVED** [review]

**Pull request merged**